doc/_templates/layout.html

```diff
@@ -23,6 +23,10 @@
         </li>
     </ol>
 </nav>
+{% elif "diffs/" in pagename %}
+<nav class="menu">
+    <button class="diff-toggle" onclick="toggleElements()">Only Show Diffs</button>
```

Comment:
```suggestion
    <button class="diff-toggle">Only Show Diffs</button>
```

Then use an `addEventListener` in the javascript.

Reply:
Will update